client/my-sites/site-settings/jetpack-site-stats.jsx

```diff
@@ -115,34 +113,34 @@ class JetpackSiteStats extends Component {
 		);
 	}
 
-	renderModuleEnableBanner() {
-		const { translate } = this.props;
+	render() {
+		const {
+			activatingStatsModule,
+			moduleUnavailable,
+			siteId,
+			siteRoles,
+			siteSlug,
+			translate,
+		} = this.props;
 
-		return (
-			<Banner
-				title={ translate( 'Site Stats module is disabled' ) }
-				description={ translate(
-					'Enable this to see detailed information about your traffic, likes, comments, and subscribers.'
+		const header = (
+			<JetpackModuleToggle
+				siteId={ siteId }
+				moduleSlug="stats"
+				label={ translate(
+					'See detailed information about your traffic, likes, comments, and subscribers.'
 				) }
-				callToAction={ translate( 'Enable' ) }
-				onClick={ this.handleStatsActivationButton }
-				event={ 'site_stats_module_enable_banner' }
-				icon="stats"
+				disabled={ activatingStatsModule || moduleUnavailable }
```

Comment:
`JetpackModuleToggle` doesn't need the `activatingStatsModule` in its `disabled` - it will check for that anyway. This will allow us to also remove the `activateModule` action creator altogether.

Reply:
🎉 d520fceb37d6b050baf54f0994d0e831c9f9b2f0